experimental/generation/generator/src/library/mergeAssets.ts

```diff
@@ -682,6 +659,19 @@ async function mergeDialogs(schemaName: string, oldPath: string, newPath: string
         }
     }
 
+    for (let trigger of newObj['triggers']) {
+        if (typeof trigger !== 'string') {
+            // todo inline object
```

Comment:
>todo [](start = 15, length = 4)

I believe if you make these todo: visual studio will pick them up.  Not sure about just todo.  Can't you just copy the object into triggers?